cli/cmds_issues.go

```diff
@@ -0,0 +1,153 @@
+
+	for _, f := range project.Data.Organization.ProjectV2.Fields.Nodes {
+		fields[f.Name] = f.Id
+		c.Printf("    <lightBlue>%s</> <> <lightCyan>%s</>\n", f.Name, f.Id)
+
+		if f.Name == "Status" {
+			for _, s := range f.Options {
+				statuses[s.Name] = s.Id
+				c.Printf("      <blue>%s</> <> <cyan>%s</>\n", s.Name, s.Id)
+			}
+		}
+	}
+	fmt.Println()
+	//For issues, only 1 repo will be passed so below will only use f.Repos[0]
+	r := gh.NewRepo(f.Owner, f.Repos[0], f.Token)
+
+	// get all issues
+	c.Printf("Retrieving all issues for <white>%s</>/<cyan>%s</>...", r.Owner, r.Name)
+	issues, err := r.GetAllIssues("open")
+
+	if err != nil {
+		c.Printf("\n\n <red>ERROR!!</> %s\n", err)
+		return nil
+	}
+	c.Printf(" found <yellow>%d</>\n", len(*issues))
+
+	filters := f.GetFilters()
+	fmt.Println("Filtering:")
+
+	//Currently not interested in the username of the author for issues, so I removed the code for now
+	totalIssues := 0
+	daysOpen := 0
+	totalDaysOpen := 0
+
+	for _, issue := range *issues {
+		issueNode := *issue.NodeID
+
+		//only put issues labeled whatever flag is passed (bug, etc) into the project, therefore graphyQL is inside this loop
+
+		for _, f := range filters {
+			match, err := f.Issue(issue)
+			if err != nil {
+				return fmt.Errorf("ERROR: running filter %s: %w", f.Name, err)
+			}
+			if match {
+				c.Printf("Syncing issue <lightCyan>%d</> (<cyan>%s</>) to project.. ", issue.GetNumber(), issueNode)
+				iid, err := p.AddToProject(pid, issueNode)
+				if err != nil {
+					c.Printf("\n\n <red>ERROR!!</> %s", err)
+					continue
+				}
+				c.Printf("<magenta>%s</>", *iid)
+
+				totalIssues++
+				daysOpen = int(time.Now().Sub(issue.GetCreatedAt()) / (time.Hour * 24))
+				totalDaysOpen = totalDaysOpen + daysOpen
+
+				//statuses and waiting days code removed
+
+				c.Printf("  open %d days\n", daysOpen)
+				q := `query=
+								mutation (
+								  $project:ID!, $item:ID!, 
+								  $issue_field:ID!, $issue_value:String!, 
+								  $daysOpen_field:ID!, $daysOpen_value:Float!, 
+								) {
+								  set_issue: updateProjectV2ItemFieldValue(input: {
+									projectId: $project
+									itemId: $item
+									fieldId: $issue_field
+									value: { 
+									  text: $issue_value
+									}
+								  }) {
+									projectV2Item {
+									  id
+									  }
+								  }
+								  set_dopen: updateProjectV2ItemFieldValue(input: {
+									projectId: $project
+									itemId: $item
+									fieldId: $daysOpen_field
+									value: { 
+									  number: $daysOpen_value
+									}
+								  }) {
+									projectV2Item {
+									  id
+									  }
+								  }
+								}
+				`
+
+				p := [][]string{
+					{"-f", "project=" + pid},
+					{"-f", "item=" + *iid},
+					{"-f", "issue_field=" + fields["Issue#"]},
+					{"-f", fmt.Sprintf("issue_value=%d", *issue.Number)},
+					{"-f", "daysOpen_field=" + fields["Open Days"]},
+					{"-F", fmt.Sprintf("daysOpen_value=%d", daysOpen)},
```

Comment:
so unless you are going to check events it might just be best to add "date opened" instead otherwise a pr that was just reopened afater being close for a long time will pop to the top

Reply:
I see what you mean, I am interested in keeping track of days since it was created, I will rename this column to clarify what the number means.